### PR TITLE
Updated Management Entity and added a user-friendly ManagementAgentService to use it

### PR DESCRIPTION
**Pull request by mathieucarbou**
__DEPENDS ON__ Terracotta-OSS/terracotta-apis#83

### CODE REVIEW

File: management-entity/management-entity-server/src/main/java/org/terracotta/management/entity/server/ManagementAgentImpl.java [hunk 1]
```diff
@@ -76,7 +76,7 @@
   }
 
   @Override
-  public Future<Void> exposeManagementMetadata(@ClientId Object clientDescriptor, ContextContainer contextContainer, Collection<Capability> capabilities) {
+  public Future<Void> exposeManagementMetadata(@ClientId Object clientDescriptor, ContextContainer contextContainer, Capability... capabilities) {
```
Comment (anthonydahanne) [inline management-entity/management-entity-server/src/main/java/org/terracotta/management/entity/server/ManagementAgentImpl.java]: weird, here you use varargs (...) where as in ManagementAgent you used []

File: management-entity/management-entity-tests/src/test/java/org/terracotta/management/entity/ManagementEntityTest.java [hunk 2]
```diff
@@ -83,37 +84,31 @@ public ContextContainer getContextContainer() {
     registry.register(new MyObject("myCacheManagerName", "myCacheName2"));
 
     try (Connection connection = stripeControl.createConnectionToActive()) {
-      EntityRef<ManagementAgentEntity, ManagementAgentConfig> ref = connection.getEntityRef(ManagementAgentEntity.class, Version.LATEST.version(), "SUPER_NAME");
-      ref.create(new ManagementAgentConfig());
-      ManagementAgentEntity entity = ref.fetchEntity();
 
-      ClientIdentifier clientIdentifier = entity.getClientIdentifier(null).get();
+      ManagementAgentService managementAgent = new ManagementAgentService(connection);
```
Comment (anthonydahanne) [inline management-entity/management-entity-tests/src/test/java/org/terracotta/management/entity/ManagementEntityTest.java]: maybe rename this test with ManagementAgentServiceTest ?

**Comment by anthonydahanne** [comment]
I know it's gonna beak, but cloudbees, test this please

File: management-entity/management-entity-tests/src/test/java/org/terracotta/management/entity/ManagementEntityTest.java (hunk 2 above)
Comment (mathieucarbou) [inline management-entity/management-entity-tests/src/test/java/org/terracotta/management/entity/ManagementEntityTest.java]: yes 👍

**Comment by mathieucarbou** [comment]
test this please

**Comment by mathieucarbou** [comment]
test this please

**Pull request merged**